src/pages/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from "react";
+import { firstResponse } from "../redux/actions/response";
+import { connect } from "react-redux";
+import { AppState } from "../redux/reducers/state";
+import Mouth from "../components/Mouth";
```

Comment:
```suggestion
import Mouth from "../components/mouth";
```

Reply:
This fixes the error :smile:

Reply:
I renamed the file to `Mouth.jsx` because React components are PascalCase by default. It didn't break on my local because my Mac has a case-insensitive file system because of Photoshop requirement. Good spotting!

Reply:
Looks good. Working now :tada: